runtime/doc/filetype.txt

```diff
@@ -542,6 +542,17 @@ CTRL-T                    Jump back to the location that the manpage was
                           opened from.
 q                         :quit if invoked as $MANPAGER, otherwise :close.
 
+Global mappings:
+K                         Jump to the manpage for the <cword> under the
+                          cursor. Takes a count for the section.
+
+If you want to define a mapping on another key, do: >
+    :nmap <silent> K :Man<CR>
+
+Replace K with your key and you can include modifiers so if you want
+it to open the page in a vertical split: >
+    :nmap <silent> K :vertical Man<CR>
```

Comment:
This is mentioned on line 536. 

Defining a mapping is a basic usage of the editor. It seems like an anti-pattern to explain basic operations over and over in every single plugin doc. If there's some piece of essential information here, sure, we should mention it. But not defining a mapping...